File: php/Processor.php
<?php

class Processor
{
    private $name = "";
    private $price = 0;

    # Constructor
    public function __construct($n="", $p=0)
    {
        $this->name = $n;
        $this->price = $p;
    }

    # Setter and Getter
    public function setName($na="")
    {
        $this->name = $na;
    }

    public function getName()
    {
        return $this->name;
    }

    public function setPrice($p=0)
    {
        $this->price = $p;
    }

    public function getPrice()
    {
        return $this->price;
    }

    # Menampilkan isi kelas
    public function showProcessor()
    {
        echo "Processor Name  : " . $this->name . "<br>";
        echo "Processor Price : Rp. " . $this->price . "<br>";
    }
}

?>

File: python/Processor.py
from unicodedata import name


class Processor():

    # Constructor
    def __init__(self):
        self.name = ""
        self.price = 0
    
    # Setter and Getter
    def setName(self, name):
        self.name = name
    
    def getName(self):
        return self.name
    
    def setPrice(self, price):
        self.price = price
    
    def getPrice(self):
        return self.price
    
    # Menampilkan isi atribut
    def showProcessor(self):
        print("Processor Name  : " + str(self.name))
        print("Processor Price : Rp. " + str(self.price))